scripts/perf/env.json

```diff
@@ -14,14 +18,15 @@
       "RESTATE_METADATA_SERVER__TYPE": "replicated",
       "RESTATE_AUTO_PROVISION": "true",
       "RESTATE_ADVERTISED_ADDRESS": "http://n1:5122",
-      "RESTATE_WORKER__INVOKER__CONCURRENT_INVOCATIONS_LIMIT": "256",
       "RESTATE_WORKER__INVOKER__INACTIVITY_TIMEOUT": "1sec",
```

Comment:
If there is no limit, then we probably can remove the reduction of the inactivity timeout.

Reply:
Good point. I will update it.